NOTICE.txt

```diff
@@ -0,0 +1,15 @@
+==========
```

Comment:
minor nitpick: I think we should name this file `NOTICE`, for consistency with the existing LICENSE.